package.json

```diff
@@ -13,8 +13,8 @@
     "csslint": "stylelint **/*.css **/*.vue",
     "csslintfix": "stylelint **/*.css **/*.vue --fix",
     "dev": "tsc --watch",
-    "install": "npm run install:hooks",
-    "install:hooks": "git config core.hooksPath .githooks",
+    "install:hooks": "python3 ./pre-commit/pre-commit-2.20.0.pyz install",
+    "uninstall:hooks": "python3 ./pre-commit/pre-commit-2.20.0.pyz uninstall",
```

Comment:
Would remove these 2 npm commands completely (see `settingUp.md` comment for rationale)